File: JCC.Domain/Entities/Projection.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JCC.Domain.Entities
{
    public class Projection
    {
        [Key]
        public int ProjectionId { get; set; }
        public DateTime horaire{get; set;}
        //foreign Key properties
        public int? CinemaId { get; set; }
        public virtual cinema Cinema { get; set; }
        //foreign Key properties
        public int? FilmId { get; set; }
        public virtual Film Film { get; set; }
        public virtual ICollection<Ticket> Tickets { get; set; }
}
}

File: JCC/Models/Projection.vb
Using System;
Using System.Data.Entity;
Using System.ComponentModel.DataAnnotations;
Using System.ComponentModel.DataAnnotations.Schema;
Namespace JCC.Models
    Public Class Projection{
    Public int id {Get;Set;}
    Public Time horaire{Get; Set;}
    Public virtual Cinema Cinema { Get; Set; }
    Public virtual Film Film{ Get; Set; }

            }